sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/SonarAnalyzer.CSharp.nuspec

```diff
@@ -10,9 +10,9 @@
     <projectUrl>http://vs.sonarlint.org</projectUrl>
     <icon>images\sonarlint_64.png</icon>
```

Comment:
If we don't use it anymore for SonarLint, I suggest to remove the references to the product.
We could use:
- for the projectUrl: https://redirect.sonarsource.com/plugins/csharp.html
- for the icon: the SonarSource icon

Reply:
RE: the project URL, after a talk with @ganncamp , we've decided to keep the SL URL, because ideally users will start using the SonarLint extension in their IDE instead of consuming the nuget.

Reply:
Fine by me